spec/models/project_spec.rb

```diff
@@ -7,4 +7,21 @@
     it {should have_many :contestant_projects}
     it {should have_many(:contestants).through(:contestant_projects)}
   end
+
+  describe '#instance method' do
+    it 'counts number of contestants on a project' do
+      challenge_1 = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+
+      project_1 = challenge_1.projects.create(name: "News Chic", material: "Newspaper")
+      project_2 = challenge_1.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+      contestant_1 = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+
+      ContestantProject.create(contestant_id: contestant_1.id, project_id: project_1.id)
+      ContestantProject.create(contestant_id: contestant_1.id, project_id: project_2.id)
+      
+      expect(project_1.contestant_count).to eq(1)
+    end
+  end
+
```

Comment:
I feel like I'm repeating myself but it's worth repeating, make the tests more robust. For every instance method you test, you should have at least 2 assertions minimum.